prosessering-core/src/main/kotlin/no/nav/familie/prosessering/internal/TaskWorker.kt

```diff
@@ -96,13 +99,39 @@ class TaskWorker(
     fun rekjørSenere(taskId: Long, e: RekjørSenereException) {
         log.info("Rekjører task=$taskId senere, triggerTid=${e.triggerTid}")
         secureLog.info("Rekjører task=$taskId senere, årsak=${e.årsak}", e)
-        val taskMedNyTriggerTid = taskService.findById(taskId)
-            .medTriggerTid(e.triggerTid)
-        taskService.klarTilPlukk(
-            taskMedNyTriggerTid,
-            endretAv = BRUKERNAVN_NÅR_SIKKERHETSKONTEKST_IKKE_FINNES,
-            melding = e.årsak,
-        )
+
+        val taskMedNyTriggerTid = taskService.findById(taskId).medTriggerTid(e.triggerTid)
```

Comment:
Når denne ble lagt til så var tanken noe i stilen med at vi ønsker en type "Exception" som ikke er en feil, og at man ønsker å rekjøre senere. EF håndterer feil hvis antallFeil er over X, kast annen feil enn `RekjørSenere..`

Føler det ble mye kode som i prinsipp det samme som i `doFeilhåndtering`, så kanskje er bedre å slå sammen disse? Hvis man slår de sammen, eks med 
`val triggerTid = if(e is RekjørSenereException) e.triggerTid else task.triggerTid.plusSeconds(finnTriggerTidVedFeil(task.type))`

og
```kotlin
private fun getStacktraceAsString(cause: Throwable?): String? {
            if (cause == null || cause is TaskExceptionUtenStackTrace || cause is RekjørSenereException) {
```

Så blir forskjellen at det blir logget en feil-logg, når man har en "RekjørSenere", som kanskje er greit? 
Samtidig var tanken bak RekjørSenere å ikke være en type feil

Reply:
Ah - ser poenget ditt @blommish

Reply:
Godt forslag!

Jeg merger denne pullrequesten, men har laget et favrokort for denne refaktoreringen som jeg setter på meg -
Da får vi en separat PR som tar for seg endringene slik at vi ikke fikser to ting samtidig.